pulpcore/cli/core/orphan.py

```diff
@@ -16,13 +18,39 @@ def orphan() -> None:
 
 
 @orphan.command()
+@click.option(
+    "--content-hrefs",
+    help=_("List of specific Contents to delete if they are orphans"),
+    callback=load_json_callback,
+)
+@click.option(
+    "--protection-time",
+    type=int,
+    help=_(
+        "How long in minutes Pulp should hold orphan Content and Artifacts before becoming"
+        " candidates for cleanup task"
+    ),
+)
 @pass_pulp_context
-def cleanup(pulp_ctx: PulpContext) -> None:
+def cleanup(
+    pulp_ctx: PulpContext, content_hrefs: Optional[List[str]], protection_time: Optional[int]
+) -> None:
     """
     Cleanup orphaned content.
     """
+    body: Dict[str, Any] = {}
+    if content_hrefs is not None:
+        pulp_ctx.needs_plugin(
+            PluginRequirement("core", "3.14", feature=_("the content_hrefs option"))
+        )
+        body["content_hrefs"] = content_hrefs
+    if protection_time is not None:
+        pulp_ctx.needs_plugin(
+            PluginRequirement("core", "3.15", feature=_("the protection_time option"))
+        )
```

Comment:
I think this can be more elegant when using a pulp_option.

Reply:
I tried that, it needed an entity context object. Should I make one for the command?

Reply:
I see. Then it's probably not worth it.

Is the entity context only used for the help text? We can probably opt out of that...